frontend/testing/playwright/tests/invalid-login/invalid-login.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from '../../extenders/testExtend';
+import { LoginPage } from '../../pages/LoginPage';
+import { expect } from '@playwright/test';
+import { Language } from '../../enum/Language';
+import { waitFor } from '@testing-library/dom';
+
+test('It is not possible to log in with invalid credentials', async ({ page }): Promise<void> => {
+  const loginPage = new LoginPage(page);
+
+  await loginPage.goToAltinnLoginPage();
+  await loginPage.goToGiteaLoginPage();
+
+  const lang = await loginPage.getLanguage();
+  if (lang !== Language.Norwegian) {
+    await loginPage.clickOnLanguageMenu();
+    await loginPage.clickOnNorwegianLanguageOption();
+    await waitFor(() => expect(loginPage.getLanguage()).toBe(Language.Norwegian));
+  }
```

Comment:
Should this be a separate helper method maybe?

Reply:
@mlqn have a frontend pr pending. If needed we can fix the comment there.